epochStart/metachain/trigger.go

```diff
@@ -11,13 +11,20 @@ import (
 	"github.com/ElrondNetwork/elrond-go/epochStart"
 )
 
+// EpochStartNotifier defines which actions should be done for handling new epoch's events
```

Comment:
Is there any reason why this interface was moved from the epochstart/interface in both epochstart/metachain/trigger and epochstart/shardchain/trigger? It works both ways

Reply:
done moved to interface.go

Reply:
It was moved back, removing duplication

Reply:
ok